src/App.tsx

```diff
@@ -1,10 +1,22 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [key, setKey] = useState('');
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+  useEffect(() => {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      setKey(event.key);
```

Comment:
![image](https://github.com/mate-academy/react_keyboard-hooks/assets/96409563/453e6392-40b3-4c9d-9f4c-431f2020be95)

Reply:
Done, than you for your review